feature-libs/user/profile/components/register/register-consents.service.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import { UntypedFormArray, UntypedFormBuilder } from '@angular/forms';
+import { ConsentTemplate } from '@spartacus/core';
+
+@Injectable()
+export class RegisterConsentsService {
+  constructor(protected fb: UntypedFormBuilder) {}
+
+  loadAdditionalConsents(): {
```

Comment:
Maybe it's only me.... The word `load` suggest that the operation will be asynchronous (so I'd expect Observable in the return type). Do you think that word `get` would be more appropriate?

PS. Did you consider simply `loadConsents` or `getConsents` (without the word "additional"). Similar question applies to another method in this class - did you consider removing the word "additional"? There are not "standard" consents in this class I guess (maybe I'm wrong?), so that's why I'm confused we had a word "additional" ;-)

Reply:
1. 'load' is changed to 'get'
2. The reason why 'Additional Consents' was used instead of just 'Consents' because - we already have 'Anonymous Consents' and 'terms and conditions' - so just to make it even more clear that these are "Additional" ones, I used this name. is it fine ?

Reply:
Thanks. Understood. No need to change